src/Gaffer/MetadataAlgo.cpp

```diff
@@ -62,6 +62,20 @@ size_t findNth( const std::string &s, char c, int n )
 	return result;
 }
 
+IECore::InternedString g_connectionColorKey( "connectionGadget:color" );
+IECore::InternedString g_noduleColorKey( "nodule:color" );
+
+void copy( const Gaffer::GraphComponent *src , Gaffer::GraphComponent *dst , IECore::InternedString key , Gaffer::MetadataAlgo::Overwrite overwrite )
+{
+	if (Gaffer::MetadataAlgo::KeepExisting == overwrite && Gaffer::Metadata::value<IECore::Data>(dst, key))
```

Comment:
Braces even for one-line conditionals please - I do hate the vertical space wastage but when context switching between Python and C++ all day long I think it's more important that we keep the two forms of blocking distinct.

Edit : looks like you already took care of this in a later commit - thanks.